src/util-lua-hassh.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright (C) 2017 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+/**
+ * \file
+ *
+ * \author Vadym Malakhatko <v.malakhatko@sirinsoftware.com>
+ */
+
+#ifndef __UTIL_LUA_HASSH_H__
+#define __UTIL_LUA_HASSH_H__
+
+#ifdef HAVE_LUA
+
+int LuaRegisterHasshFunctions(lua_State *luastate);
+
+#endif /* HAVE_LUA */
+
+#endif /* __UTIL_LUA_HASSH_H__ */
```

Comment:
wonder what that symbol at the end of this line means. Any idea?

Reply:
Can you, please, point to the exact symbol, because I didn't understand the problem here.